packages/web/src/shared/AwesomeFade.tsx

```diff
@@ -0,0 +1,72 @@
+import { keyframes } from '@emotion/core'
+import * as React from 'react'
+import Reveal from 'react-awesome-reveal'
+
+type Direction = 'up' | 'right'
+
+interface Props {
+  children: React.ReactNode
+  delay?: number
+  distance?: number | string
+  duration?: number
+  fraction?: number
+  reverse?: boolean
+  direction?: Direction
+}
+
+export default React.memo(function AwesomeFade({
+  children,
+  reverse,
+  delay,
+  distance,
+  direction,
+  duration,
+  fraction,
```

Comment:
What does "Fraction" do?

Reply:
it's passed to the intersection observer. Its the fraction of an element that needs to be visible before it is considered onscreen